downstream/modules/platform/con-eda-2-5-with-controller-2-4.adoc

```diff
@@ -0,0 +1,59 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-07-05
+
+:_mod-docs-content-type: CONCEPT
+
+[id="eda-2-5-with-controller-2-4_{context}"]
+= {EDAName} 2.5 with controller 2.4
+
+Use the following example to populate the inventory file to deploy a new single instance of {EDAName} 2.5 with controller 2.4. For {EDAName}, the requirements for a connection to controller are the `automation_controller_main_url` pointing to the 2.4 controller URL and `username/password`.
```

Comment:
```suggestion
Use the following example to populate the inventory file to deploy a new single instance of {EDAName} 2.5 with controller 2.4. For {EDAName}, the requirements for a connection to controller are the `automation_controller_main_url` pointing to the 2.4 controller URL.
```

If we want to mention authentication credentials for controller, then we should link to: https://docs.ansible.com/automation-controller/latest/html/userguide/users.html#users-tokens otherwise its not needed in the installation inventory file at this time.